apps/docs/src/packages/headless.md

```diff
@@ -0,0 +1,58 @@
+---
+outline: [2, 3]
+---
+
+<script lang="ts" setup>
+import packageJson from "../../../../packages/headless/package.json";
+</script>
+
+# @sit-onyx/headless
+
+::: warning Work in progress / Active development
+This library is currently in early / active development.
+:::
+
+{{ packageJson.description }}.
+
+Inspired by [Melt UI](https://melt-ui.com).
+
```

Comment:
we don't have it yet in npmjs.com right? as soon as we have it, we might want to add the link to the package here :)

Reply:
Not yet, lets do this after they are released :)